### ISSUE TITLE
myjenkins: Install latest docker release

### ISSUE THREAD

**Issue by gmacario** [gmacario/easy-jenkins#35]
According to [the results when building project test_docker](https://github.com/gmacario/easy-jenkins/blob/master/docs/building-easyjenkins-inside-easyjenkins.md), docker 1.9.1 was installed in service `myjenkins`, even though docker 1.10.0 was already released:

```
Started by user anonymous
[EnvInject] - Loading node environment variables.
Building in workspace /var/jenkins_home/workspace/test_docker
[test_docker] $ /bin/bash -xe /tmp/hudson5476115026167153721.sh
+ id
uid=0(root) gid=0(root) groups=0(root)
+ pwd
/var/jenkins_home/workspace/test_docker
+ docker --version
Docker version 1.9.1, build a34a1d5
+ docker version
Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 12:59:02 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 12:59:02 UTC 2015
 OS/Arch:      linux/amd64
+ docker info
...
```

**Comment by gmacario** [comment]
From https://github.com/gmacario/easy-jenkins/blob/master/myjenkins/Dockerfile#L24-L25

```
# Install Docker
RUN wget -qO- https://get.docker.com/ | sh
```

However it looks like the script at https://get.docker.com/ is not up-to-date, cfr. https://github.com/docker/docker/blame/master/hack/install.sh

**Comment by gmacario** [comment]
As of 2016-02-07 11:22 CET the probably seems gone away.

Result of build `test_docker`

```
Started by user anonymous
[EnvInject] - Loading node environment variables.
Building in workspace /var/jenkins_home/workspace/test_docker
[test_docker] $ /bin/bash -xe /tmp/hudson729394599786323653.sh
+ id
uid=0(root) gid=0(root) groups=0(root)
+ pwd
/var/jenkins_home/workspace/test_docker
+ docker --version
Docker version 1.10.0, build 590d5108
+ docker version
Client:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 18:16:19 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.10.0
 API version:  1.22
 Go version:   go1.5.3
 Git commit:   590d5108
 Built:        Thu Feb  4 18:16:19 2016
 OS/Arch:      linux/amd64
+ docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 0
Server Version: 1.10.0
Storage Driver: devicemapper
 Pool Name: docker-0:32-181-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 10.74 GB
 Backing Filesystem: ext4
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 305.7 MB
 Data Space Total: 107.4 GB
 Data Space Available: 47.74 GB
 Metadata Space Used: 729.1 kB
 Metadata Space Total: 2.147 GB
 Metadata Space Available: 2.147 GB
 Udev Sync Supported: false
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 WARNING: Usage of loopback devices is strongly discouraged for production use. Either use `--storage-opt dm.thinpooldev` or use `--storage-opt dm.no_warn_on_loop_devices=true` to suppress this warning.
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.90 (2014-09-01)
Execution Driver: native-0.2
Logging Driver: json-file
Plugins: 
 Volume: local
 Network: null host bridge
Kernel Version: 4.1.17-boot2docker
Operating System: Debian GNU/Linux 8 (jessie) (containerized)
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.956 GiB
Name: c2eb63618350
ID: UIQS:A2YO:DHQU:EUQZ:S2N7:ZVIK:4IU3:MHYM:FE6G:J652:72LH:Z6SI
WARNING: bridge-nf-call-iptables is disabled
WARNING: bridge-nf-call-ip6tables is disabled
+ docker-compose --version
docker-compose version 1.5.2, build 7240ff3
Notifying upstream projects of job completion
Finished: SUCCESS
```

Configuration: https://github.com/gmacario/easy-jenkins master running in docker-machine `easy-jenkins` on Oracle VM VirtualBox 5.0.14 r105127.

Closing issue
